webpack.config.js

```diff
@@ -17,8 +17,8 @@ module.exports = {
             test: /\.js?$/,
             loader: 'babel',
             query: { 
-              presets: ['react', 'es2015'], 
-              plugins: ['transform-es2015-modules-commonjs', 'transform-object-rest-spread'] 
+              presets: ['react', 'stage-0', 'es2015'], 
+              plugins: ['transform-runtime'] 
```

Comment:
IE and some android webview doesn't support Object.assign
transform-runtime will fix it